src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -424,21 +424,38 @@ private EmailWrapper generateFeedbackSessionEmailBaseForStudents(
         return email;
     }
 
+    private String generateInstructorFragment(String courseId, String courseName) {
+
+        String fragmentParameters;
+
+        /* If the course name == "", we understand that this method was called by the
+        generateFeedbackSessionEmailBaseForInstructors, which does not need the course name */
```

Comment:
I do not know if you would like to add it for the `generateFeedbackSessionEmailBaseForInstructors` as well, but in the context of the method that this issue is related to `generateFeedbackSessionClosedEmails`, I thought it would be good to include both the id and the name. Do you want to add it for the other method as well?